### PR TITLE
DOMA-1186 changed "specializations" field appearance on presence of `canBeAssignedAsExecutor` role ability

### PR DESCRIPTION
**Pull request by AntonAL**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/organization/components/EmployeeForm/CreateEmployeeForm.tsx [hunk 1]
```diff
@@ -208,9 +207,10 @@ export const CreateEmployeeForm: React.FC = () => {
                                                             const selectedRole = find(employeeRoles, { id: role })
 
                                                             const isDivisionsEnabled = hasFeature('division')
+                                                            console.debug(selectedRole)
```
Comment (pahaz) [inline apps/condo/domains/organization/components/EmployeeForm/CreateEmployeeForm.tsx:210]: Debug

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/organization/components/EmployeeForm/UpdateEmployeeForm.tsx [hunk 2]
```diff
@@ -167,8 +167,9 @@ export const UpdateEmployeeForm = () => {
                                                 ({ getFieldValue })=> {
                                                     const role = getFieldValue('role')
                                                     const selectedRole = find(employeeRoles, { id: role })
+                                                    console.debug(selectedRole)
```
Comment (Dimitreee) [inline apps/condo/domains/organization/components/EmployeeForm/UpdateEmployeeForm.tsx:170]: remove logging

File: apps/condo/domains/organization/components/EmployeeForm/CreateEmployeeForm.tsx (hunk 1 above)
Comment (AntonAL) [reply apps/condo/domains/organization/components/EmployeeForm/CreateEmployeeForm.tsx]: Oops, removed in `2d17502e`

File: apps/condo/domains/organization/components/EmployeeForm/UpdateEmployeeForm.tsx (hunk 2 above)
Comment (AntonAL) [reply apps/condo/domains/organization/components/EmployeeForm/UpdateEmployeeForm.tsx]: Removed in `2d17502e`

**Review by toplenboren — APPROVED** [review]

**Pull request merged**